llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/Input/InputDialect.h

```diff
@@ -16,4 +16,20 @@
 #define GET_TYPEDEF_CLASSES
 #include "iree-dialects/Dialect/Input/InputTypes.h.inc"
 
+//===----------------------------------------------------------------------===//
+// IREE ABI helpers for constructing buffer views
+//===----------------------------------------------------------------------===//
+
+namespace mlir::iree_compiler::IREE::Input {
+
+// Returns a stable identifier for the MLIR element type or nullopt if the
+// type is unsupported in the ABI.
+std::optional<int32_t> getElementTypeValue(Type type);
+
+// Returns a stable identifier for the MLIR encoding type or 0 (opaque) if the
```

Comment:
0 or nullopt ?

Reply:
Right now it's always 1, but the intention to return empty optional